### DISCUSSION TITLE
`fig.path`

### DISCUSSION THREAD

**Discussion by cgoo4** [quarto-dev/quarto-cli#2746] [Q&A] [answered]
In Rmarkdown it's possible to save a chunk-generated plot with `fig.path=`. 

The below approach works in a qmd. Could `fig-path` be added as a hashpipe option?

```
```{r, fig.path=""}
#| label: feature
#| fig-height: 4.83
#| fig-width: 9
```
```

**Comment by mcanouil** [comment, marked as answer]
The hashpipe comes from `knitr` in your case, all `knitr` options should work using yaml hashpipe style.

May I suggest to switch this discussion category from "Ideas" to "Q&A" and select this answer if appropriate? (From the desktop version of GitHub website)

````md
---
title: "Untitled"
format: html
---

```{r}
#| label: feature
#| fig-height: 4.83
#| fig-width: 9
#| fig-path: "subdir/"
plot(1)
```
````

It works perfectly.
<img width="1624" alt="Screenshot of VSCode showing the project directory tree structure with a figure in a directory called subdir, a Quarto document with a R code cell using the fig-path option set to subdir and the rendered HTML file on the right" src="https://user-images.githubusercontent.com/8896044/194717716-21df131d-1077-46ba-98e5-c5a24a237b1c.png">

**Comment by cgoo4** [comment]
Perfect, thank you. (I thought I had attempted that, but must have misstepped.)

I'm wondering though if it could be added to the suggested options?

<img width="500" alt="image" src="https://user-images.githubusercontent.com/32419319/194719262-9ee47e97-d7cc-4d1d-9c83-9b7a4105a7e2.png">

(The options don't all display alphabetically, which would also be good, but I couldn't see `fig-path` in the full list.)

**Reply by mcanouil** [reply]
This would be an issue to submit to RStudio repository and/or to Quarto VSCode extension repository.

**Comment by cgoo4** [comment]
Added an issue [here](https://github.com/rstudio/rstudio/issues/12122#issue-1402240884).
